nixos/modules/programs/light.nix

```diff
@@ -63,13 +72,14 @@ in
         let
           light = "${pkgs.light}/bin/light";
           step = builtins.toString cfg.brightnessKeys.step;
+          minBrightness = builtins.toString cfg.brightnessKeys.minBrightness;
         in
         [
           {
             keys = [ 224 ];
             events = [ "key" ];
-            # Use minimum brightness 0.1 so the display won't go totally black.
-            command = "${light} -N 0.1 && ${light} -U ${step}";
+            # -N is used to ensure that value >= minBrightness
+            command = "${light} -N ${minBrightness} && ${light} -U ${step}";
```

Comment:
Would it be a good idea to prevent value under 0.1 ?

Reply:
Well isn't the `types.numbers.between 0 100` made for that?

Reply:
And if you are talking about values like 0.001 I really think this is up to the user

Reply:
> Well isn't the `types.numbers.between 0 100` made for that?

This is precisely why I wrote my comment. I have no clue yet what are the allowed values for the `numbers` type.

Reply:
Okay no problem.
So numbers.between means either int or float, and 0 <= value <= 100, so 0.00001, 0, 100.0 and 50 are allowed values.

Reply:
> numbers.between means either int or float, and 0 <= value <= 100

Where do you see this in the documentation?

Reply:
In Nix the documentation is often the source code 😂
https://github.com/NixOS/nixpkgs/blob/master/lib%2Ftypes.nix#L431-L443

Reply:
Perfect, thank you !